README.md

```diff
@@ -268,7 +268,7 @@ that will help us track down the bug faster.
 ### Submitting a Feature Request
 
 For feature requests, please feel free to file a
-[GitHub issue](https://github.com/apple/swift-docc/issues/new?template=FEATURE_REQUEST.md)
+[GitHub issue](https://github.com/apple/swift-docc/issues/new?assignees=&labels=enhancement&template=FEATURE_REQUEST.yml)
```

Comment:
I renamed the label to `enhancement`. I think that better aligns with the way the label is used elsewhere and fixes the broken link in GitHub's Issues UI.


Maybe we can simplify this a bit?

```suggestion
[GitHub issue](https://github.com/apple/swift-docc/issues/new?labels=enhancement&template=FEATURE_REQUEST.yml)
```

Reply:
Currently, the "assingess=" is just an empty query component. But that's exactly what the user will direct to when they click "Issues" -> "New Issue" -> "Feature Request Get Started".

If you prefer, we can omit it.

Reply:
I think it's fine as is!